src/ranged.cpp

```diff
@@ -154,6 +154,9 @@ int player::gun_engagement_moves( const item &gun, int target, int start ) const
 
 bool player::handle_gun_damage( item &it )
 {
+    // below are the double and int versions of the dirt_max variable. Both should be the same. Some parts of the script require an integer. Others require a double.
```

Comment:
> Both should be the same.

Than write that in code, not as a comment:
```C++
const double dirt_max_dbl = dirt_max_int;
```
Or even cast the `int` to `double` whenever you need it to be a double.

Reply:
> > Both should be the same.
> 
> Than write that in code, not as a comment:
> 
> ```c++
> const double dirt_max_dbl = dirt_max_int;
> ```
> 
> Or even cast the `int` to `double` whenever you need it to be a double.

So what's up is that good? Don't want this to drag on too long and get mired in unresolved conflicts.